### DISCUSSION TITLE
Bold font in website title

### DISCUSSION THREAD

**Discussion by samrit2442** [quarto-dev/quarto-cli#9327] [Q&A] [answered]
### Description

I want to add a bold font style to my quarto website title from the scss file. How can I do that? I am able to make a bold font for the navigation bar using the code below but not the title.
```{scss}
.nav-link {
  font-weight: bold;
}
```

**Comment by mcanouil** [comment, marked as answer]
You need to use your favourite browser developer/inspect mode to see what you should target, then apply that using `theme` (SCCS), `css` (CSS) or as raw HTML in `include-*`.

Your question is unfortunately a pure HTML/CSS question which falls a bit outside of Quarto's scope.

**Reply by samrit2442** [reply]
Yes, i am able to do that with class "navbar-title". Thank you.
